stacks/components/users/templates/users.yaml

```diff
@@ -0,0 +1,68 @@
+users:
+  - name: "{{ username }}"
+    alterableOnly: {{ alterableOnly }}
+    tags: {{ tags }}
+    properties:
+      {% if display_name is defined and display_name != 'null' %}
```

Comment:
@paladin235 How do these work with the new null checks now? Is the if statement needed?

Reply:
You should just be able to do `DISPLAY_NAME: {{ display_name }}` and call it a day as long as `display_name` is defined explicitly as null or empty string in the group defaults.

Reply:
I am not 100% sure how I got stuck in that line of thinking, I know it centered around password initial... in either case, it's updated and attributed to PEBKAC.